src/gui/tag/TagsEdit.cpp

```diff
@@ -797,6 +799,15 @@ void TagsEdit::keyPressEvent(QKeyEvent* event)
     } else if (event == QKeySequence::SelectNextChar) {
         impl->moveCursor(impl->text_layout.nextCursorPosition(impl->cursor), true);
         event->accept();
+    } else if (event == QKeySequence::Paste) {
+        auto clipboard = QApplication::clipboard();
+        if (clipboard) {
+            for (auto tagtext : clipboard->text().split(",")) {
```

Comment:
Shouldn't it use the same regex as when editing manually, the one on `Entry.cpp` line 195?
Also, what happens when you paste a tag that was already added?

Reply:
Tags don't duplicate so non issue. I am just trying to get this out the door right now. This needs far more work to synch the various pieces. Not doing that right now.